src/pybind/mgr/dashboard/frontend/src/app/shared/components/delete-confirmation-modal/delete-confirmation-modal.component.html

```diff
@@ -77,13 +81,15 @@
                         [form]="deletionForm"
                         [submitText]="(actionDescription | titlecase) + ' ' + itemDescription"
                         [modalForm]="true"
+                        [disabled]="(submitDisabled$ | async) || deletionForm.disabled"
                         [submitBtnType]="(actionDescription === 'delete' || actionDescription === 'remove') ? 'danger' : 'primary'"></cd-form-button-panel>
 
 </cds-modal>
 
 <ng-template #deletionHeading>
   <h3 cdsModalHeaderHeading
+      [ngClass]="bodyContext?.customHeaderClass"
       i18n>
-    {{ actionDescription | titlecase }} {{ itemDescription }}
+    {{ bodyContext?.customHeader || (actionDescription | titlecase) + ' ' + itemDescription }}
```

Comment:
There is  label prop on modal if required that can be used instead
```suggestion
				<h2 cdsModalHeaderLabel>Label</h2>
```

Reply:
changes are made